staff/views/applications.py

```diff
@@ -231,7 +229,7 @@ def get_queryset(self):
         if user := self.request.GET.get("user"):
             qs = qs.filter(created_by__username=user)
 
-        return fetch(qs.distinct())
+        return list(qs.distinct())
```

Comment:
Would a variable help in making this code more explicit? e.g

```
evaluated_qs = list(qs.distinct())
return evaluated_qs
```

I'd like to avoid a code comment if I can find an alternative, but if not, I'll add a comment.

Reply:
On reflection, I think this is just something that a Djangonaut should know. No need for a comment or a new variable.